batch/batch/worker/flock.py

```diff
@@ -3,40 +3,29 @@
 import argparse
 import subprocess as sp
 
+from pathlib import Path
```

Comment:
The path manipulations are simpler in this version than they were in my first draft. I'm not sure this is needed, and `os.path` may be fine.